### PR TITLE
Scalastyle should also check sources in test and it

### PR DESCRIPTION
**Pull request by andrewsmartin**
Also fixed some outdated headers and line length issues that were caught by this.

### CODE REVIEW

File: scio-bigquery/src/it/scala/com/spotify/scio/bigquery/BigQueryClientIT.scala [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Spotify AB.
+ * Copyright 2017 Spotify AB.
```
Comment (andrewsmartin) [inline scio-bigquery/src/it/scala/com/spotify/scio/bigquery/BigQueryClientIT.scala]: Just realized the date probably shouldn't change, just remove the (c)?

File: scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala [hunk 2]
```diff
@@ -57,7 +57,8 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
     """.stripMargin)
   class ArtisanalMoreThan22Fields
 
-  "BigQueryType.fromSchema" should "support .schema in companion object with more than 22 fields" in {
+  "BigQueryType.fromSchema".should("support .schema in companion object with more than 22 fields")
```
Comment (nevillelyh) [inline scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala]: Shorten the 2nd string so we don't have to line break and reduce readability?
`with >22 fields`?

File: scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala [hunk 3]
```diff
@@ -71,7 +72,8 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
                          a11:Int,a12:Int,a13:Int,a14:Int,a15:Int,a16:Int,a17:Int,a18:Int,a19:Int,
                          a20:Int,a21:Int,a22:Int,a23:Int)
 
-  "BigQueryType.toTable" should "not provide .tupled in companion object with more than 22 fields" in {
+  "BigQueryType.toTable".should("not provide .tupled in companion object with more than 22 fields")
```
Comment (nevillelyh) [inline scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala]: `with >22 fields`?

File: scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala [hunk 4]
```diff
@@ -80,10 +82,12 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
   }
 
   it should "support .fromTableRow in companion object" in {
-    (classOf[(TableRow => TwentyThree)] isAssignableFrom TwentyThree.fromTableRow.getClass) shouldBe true
```
Comment (nevillelyh) [inline scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala]: add a `val cls = classOf[]` to keep this in one line?

File: scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala [hunk 5]
```diff
@@ -80,10 +82,12 @@ class TypeProviderTestScala211 extends FlatSpec with Matchers {
...
+    (classOf[(TableRow => TwentyThree)]
+      isAssignableFrom TwentyThree.fromTableRow.getClass) shouldBe true
   }
 
   it should "support .toTableRow in companion object" in {
-    (classOf[(TwentyThree => TableRow)] isAssignableFrom TwentyThree.toTableRow.getClass) shouldBe true
+    (classOf[(TwentyThree => TableRow)]
```
Comment (nevillelyh) [inline scio-bigquery/src/test/scala-2.11/com/spotify/scio/bigquery/types/TypeProviderTestScala211.scala]: `val cls = classOf[]`

File: scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/TypeProviderTest.scala [hunk 6]
```diff
@@ -294,7 +294,8 @@ class TypeProviderTest extends FlatSpec with Matchers {
   }
 
   it should "create companion object that is a Function subtype" in {
-    (classOf[Function5[Long, Double, Boolean, String, Instant, ToTable]] isAssignableFrom ToTable.getClass) shouldBe true
+    (classOf[Function5[Long, Double, Boolean, String, Instant, ToTable]]
```
Comment (nevillelyh) [inline scio-bigquery/src/test/scala/com/spotify/scio/bigquery/types/TypeProviderTest.scala]: `val cls = classOf[]`

**Pull request merged**